deliver/spec/upload_screenshots_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'deliver/upload_screenshots'
+require 'fakefs/spec_helpers'
+
+describe Deliver::UploadScreenshots do
+  describe "screenshot collection" do
+    include(FakeFS::SpecHelpers)
+
+    def add_screenshot(file)
+      FileUtils.mkdir_p(File.dirname(file))
+      File.open(file, 'w') { |f| f << 'touch' }
```

Comment:
Should this be binary?

Reply:
I'm not sure.. I copied this approach from `loader_spec.rb#L14` so I hope its fine